### PR TITLE
enriched readme

### PR DESCRIPTION
**Pull request by nbokulich**
Updated readme with installation/tutorial info. I am assuming that relative links should work.

### CODE REVIEW

**Review by ebolyen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,4 +4,22 @@
...
+
+This is a QIIME 2 plugin. For details on QIIME 2 and tutorials demonstrating how to use this plugin, see the [QIIME 2 documentation](https://qiime2.org/). Tutorials for this plugin can be found [here](https://docs.qiime2.org/2018.8/tutorials/longitudinal/).
+
+## API documentation
+
+API documentation can be found [here](https://docs.qiime2.org/2018.8/plugins/available/longitudinal/).
+
+## Help
+
+For user support, see the [QIIME 2 Forum](https://forum.qiime2.org). Bug reports and feature requests can also be made [via a new issue](./issues/new/choose).
```
Comment (ebolyen) [inline README.md]: Remove the `.`, you want a path relative to root.

File: README.md [hunk 2]
```diff
@@ -4,4 +4,22 @@
...
+
+## Contributing
+
+QIIME 2 is an open-source project, and we are very interested in contributions from the community. Please see the [contributing guidelines](./.github/CONTRIBUTING.md) if you would like to get involved.
```
Comment (ebolyen) [inline README.md]: Same here

**Review by ebolyen — CHANGES_REQUESTED** [review]
Missed the links

**Comment by nbokulich** [comment]
thanks @ebolyen ! I fixed the links.

**Review by ebolyen — APPROVED** [review]

**Pull request merged**